packages/eas-cli/src/commands/update/index.ts

```diff
@@ -198,7 +199,7 @@ export default class UpdatePublish extends EasCommand {
     // If a group was specified, that means we are republishing it.
     republish = republish || !!group;
 
-    const { exp, projectId, projectDir } = await getDynamicProjectConfigAsync({
+    let { exp, projectId, projectDir } = await getDynamicProjectConfigAsync({
```

Comment:
Don't use `let`

Reply:
I also prefer `const`, but Will requested that change:
> The configs fetched on L203 and L207 are now stale, so they'll need to be re-fetched.

Using `let` here is justified I think.